src/components/binding/binding-renderer.js

```diff
@@ -770,22 +829,52 @@ genotet.BindingRenderer.prototype.drawBed_ = function() {
 genotet.BindingRenderer.prototype.updateDetailHeight_ = function() {
   var numTracks = this.data.tracks.length;
   var overviewHeight = this.data.options.showOverview ?
-    this.OVERVIEW_HEIGHT * numTracks : 0;
+    this.OVERVIEW_HEIGHT_ * numTracks : 0;
   var exonsHeight = this.data.options.showExons ?
-    this.EXON_HEIGHT : 0;
+    this.EXON_HEIGHT_ : 0;
   if (this.data.bed.aggregated) {
-    this.bedHeight_ = this.BED_HEIGHT;
+    if (this.data.bed.aggregatedChanged) {
+      this.canvasHeight -= this.bedHeight_;
+      var containerHeight = this.container.height() - this.bedHeight_;
+      this.signal('updateContainerSize', {
+        containerWidth: this.container.width(),
+        containerHeight: containerHeight
+      });
+      this.signal('updateCanvasSize', {
+        canvasWidth: this.canvasWidth,
+        canvasHeight: this.canvasHeight
+      });
+    }
+    this.bedHeight_ = this.BED_HEIGHT_;
   } else {
-    this.bedHeight_ = this.data.options.showBed ?
-    (this.canvasHeight - exonsHeight - overviewHeight) *
-    this.BED_HEIGHT_PROPORTION : 0;
+    if (this.data.bed.aggregatedChanged) {
+      this.bedHeight_ = this.data.options.showBed ?
+        this.bedHeight_ + this.BED_HEIGHT_EXTENSION_ : 0;
+      this.canvasHeight += this.BED_HEIGHT_EXTENSION_;
+      var containerHeight = this.container.height() +
+        this.BED_HEIGHT_EXTENSION_;
+      this.signal('updateContainerSize', {
+        containerWidth: this.container.width(),
+        containerHeight: containerHeight
+      });
+      this.signal('updateCanvasSize', {
```

Comment:
When the resize event is fired, will the view be re-rendered/re-layouted AFTER the resize? What is the re-rendering/re-layouting mechanism involved?